src/cds_objects.cc

```diff
@@ -171,7 +163,7 @@ void CdsItem::validate() const
 
 CdsItemExternalURL::CdsItemExternalURL()
 {
-    objectType |= OBJECT_TYPE_ITEM_EXTERNAL_URL;
+    objectType = Type::EXTERNAL_URL;
```

Comment:
`::EXTERNAL_URL `: You need to set Type::ITEM | Type::EXTERNAL_URL otherwise the search for items in sql_database does not return them anymore.

Also check the usages of `isCdsItem` and `isCdsPureItem`